src/applications/financial-status-report/components/monetary/MonetaryInputList.jsx

```diff
@@ -40,15 +42,35 @@ const MonetaryInputList = props => {
   const prompt =
     'How much are each of your financial assets worth? Include the total amounts for you and your spouse.';
 
-  // removing cash as an option if the user is eligible for streamlined
-  // but the amount of cash they have is above the threshold
+  // noCashList - remove cash in hand for original asset implementation
+  //  only used to protect save in progress for forms prior to streamlinedWaiverAssetUpdate
+  const noCashList = monetaryAssets.filter(
+    asset => asset.name.toLowerCase() !== 'cash',
+  );
+
+  // noLiquidAssetsList - remove liquid assets for streamlinedWaiverAssetUpdate
+  //  this filter hides all the newly populated fields we collect in previous steps
+  const noLiquidAssetsList = noCashList.filter(
+    asset =>
+      asset.name.toLowerCase() !== 'checking accounts' &&
+      asset.name.toLowerCase() !== 'savings accounts' &&
+      asset.name.toLowerCase() !== 'cash on hand (not in bank)' &&
+      asset.name.toLowerCase() !== 'cash in a bank (savings and checkings)',
+  );
+
+  const streamlinedList = data['view:streamlinedWaiverAssetUpdate']
+    ? noLiquidAssetsList
+    : noCashList;
```

Comment:
Is monetaryAssets ever going to contain objects without a name or do we need to update the null checks?

```
  ) ?? [];

const noLiquidAssetsList = noCashList?.filter(
    asset =>
      asset?.name?.toLowerCase() !== 'checking accounts' &&
      asset?.name?.toLowerCase() !== 'savings accounts' &&
      asset?.name?.toLowerCase() !== 'cash on hand (not in bank)' &&
      asset?.name?.toLowerCase() !== 'cash in a bank (savings and checkings)',
  ) ?? [];
  ```

Reply:
When we destructure `monetaryAssets` from `assets`, we default it to an empty array. `.filter()` will return an empty array if nothing meets the conditions, or if the passed array is empty, so I think we're save w/o the `?? []`, but I added some Elivs in with the next commit on the `asset.name` just in case those go wonky somehow.